README.md

```diff
@@ -1,8 +1,14 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
```

Comment:
**Let's make your repo look more professional by updating your `README` file**.
Checkout out the below link for help, feel free to follow this [Microverse template](https://github.com/microverseinc/readme-template) from pathwright

Make sure to include Setup and Install section.